[css-masonry] Update Intrinsic Sizing Rows 003 Mix1 Expected / Ref Test Case

stwrt opened the pull request:
#### a8303551d4dbf037322efb12adb3ac702f3bdb3e
<pre>
[css-masonry] Update Intrinsic Sizing Rows 003 Mix1 Expected / Ref Test Case
<a href="https://bugs.webkit.org/show_bug.cgi?id=283060">https://bugs.webkit.org/show_bug.cgi?id=283060</a>
<a href="https://rdar.apple.com/problem/139810909">rdar://problem/139810909</a>

Reviewed by Sammy Gill.

Use Ahem font and add colors to work around font sizing issues.

* Grid 1
Remove spans.
Add extra hidden item to correctly handle min-content size.
Add sizing for grid template columns, so the 5th item is correctly placed.

* Grid 2
Remove spans.
Add extra hidden item to correctly handle min-content size.
Add sizing for grid template columns, so the 5th item is correctly placed.

* Grid 4-5
Remove spans that were causing width issues.

* Grid 6
Remove spans that were causing width issues.
Remove extra 5 not found in the main test case.

* Grid 7 - 9
Remove spans that were causing width issues.
Remove hidden items that were adding an extra column
Add sizing for grid template columns; the grid appears to overflow without it.
Add height of 3ch to the first item, so the first row is correctly sized.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/intrinsic-sizing/masonry-intrinsic-sizing-rows-003-mix1-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/intrinsic-sizing/masonry-intrinsic-sizing-rows-003-mix1-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/intrinsic-sizing/masonry-intrinsic-sizing-rows-003-mix1.html:

Canonical link: <a href="https://commits.webkit.org/286740@main">https://commits.webkit.org/286740@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/43bfc9137ba20d788add1acfb1759845cbd25332

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/76972 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/56008 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/29888 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/81520 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/28252 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/65156 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/4304 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/60327 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/18401 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/80039 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/50268 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/66073 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/40639 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/47670 "Passed tests") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/64/builds/23568 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/26578 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/68791 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/23899 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/82956 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/4353 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/2917 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/68601 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/4507 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/66046 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/67852 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/16906 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/11834 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/9917 "Passed tests") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/4299 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/4319 "Built successfully") | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/129/builds/7754 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/6078 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

sammygill commented on LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/intrinsic-sizing/masonry-intrinsic-sizing-rows-003-mix1-expected.html:
```diff
@@ -32,28 +34,54 @@
 <body>
 
 <section>
-<grid>
```
Can you double check if this specific change and the ones like this in the grids below are similar to the ones I commented on in https://github.com/WebKit/WebKit/pull/36546#discussion_r1841623428?

In this one it seems like the problem is just that the "5 5" spans 2 when it should only span 1.

stwrt committed:
[css-masonry] Update Intrinsic Sizing Rows 003 Mix1 Expected / Ref Test Case
https://bugs.webkit.org/show_bug.cgi?id=283060
rdar://problem/139810909

Reviewed by Sammy Gill.

Use Ahem font and add colors to work around font sizing issues.

* Grid 1
Remove spans.
Add extra hidden item to correctly handle min-content size.
Add sizing for grid template columns, so the 5th item is correctly placed.

* Grid 2
Remove spans.
Add extra hidden item to correctly handle min-content size.
Add sizing for grid template columns, so the 5th item is correctly placed.

* Grid 4-5
Remove spans that were causing width issues.

* Grid 6
Remove spans that were causing width issues.
Remove extra 5 not found in the main test case.

* Grid 7 - 9
Remove spans that were causing width issues.
Remove hidden items that were adding an extra column
Add sizing for grid template columns; the grid appears to overflow without it.
Add height of 3ch to the first item, so the first row is correctly sized.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/intrinsic-sizing/masonry-intrinsic-sizing-rows-003-mix1-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/intrinsic-sizing/masonry-intrinsic-sizing-rows-003-mix1-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/intrinsic-sizing/masonry-intrinsic-sizing-rows-003-mix1.html:

Canonical link: https://commits.webkit.org/286740@main
